### PR TITLE
ActuatorDisk

### PR DESCRIPTION
**Pull request by psakievich**
This is not intended to merge just yet.  I wanted to get the code out so others can take a look at it.  
Key items that need addressing:

1) change number of points and/or scale force spreading radially
2) add a multiplier for area and/or adjusting torque/power calculations

(update) I think these things should be fairly minor compared to getting plumbing correct.  So if you see any issues with the design let's be sure to discuss those first.

### CODE REVIEW

File: src/ActuatorDiskFAST.C [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+/*
+ * ActuatorDiskFAST.C
+ *
+ *  Created on: Oct 16, 2018
+ *      Author: psakiev
+ */
+
+#ifdef NALU_USES_OPENFAST
```
Comment (sayerhs) [inline src/ActuatorDiskFAST.C]: With the explicit listing of files in `CMakeLists.txt` I think we can clean up these `#ifdef`s in the C files.

Comment (psakievich) [reply src/ActuatorDiskFAST.C]: Yeah that's a good point.  They're pretty redundant.

**Comment by psakievich** [comment]
I believe I'm ready to merge this.  If anyone else want's to review it please do so soon.  Other wise I will resolve the conflicts, squash the commits and merge at the end of the week.

**Comment by sayerhs** [comment]
@psakievich The `changes` tab shows that the `wind-utils` submodule was deleted in one of the commits. Was this deliberate?

**Comment by psakievich** [comment]
no, that must have been a mistake.

**Comment by sayerhs** [comment]
@psakievich Can you please update Nalu documentation with the equations you shared yesterday with the team explaining your implementation of the actuator disk?

**Comment by psakievich** [comment]
@sayerhs sure thing

**Pull request merged**